schema/meter_provider.json

```diff
@@ -189,6 +189,9 @@
                 "without_scope_info": {
                     "type": ["boolean", "null"]
                 },
+                "without_target_info": {
+                    "type": ["boolean", "null"]
+                },
                 "with_resource_constant_labels": {
```

Comment:
This appears to have been removed from the spec. Should remove, but can open a new issue to track rather than blocking this PR.

Reply:
#404